flag/flag.go

```diff
@@ -23,7 +22,7 @@ func Init(f interface{}) {
 	}
 
 	for k, v := range m {
-		m[k] = toValue([]string{toCase(k)}, k, v)
+		m[k] = toValue([]string{strings.ToLower(k)}, k, v)
```

Comment:
The implementation of `toCase` was not optimal. Instead of getting annoyed by all the exceptions where it does not work as expected, I voted for simply lowering all strings for simplicity. We could get proper formatting by using the original struct property case or by defining JSON tags on all the structs. At this point I do not care about that at all. So fuck it.